packages/sarek/sarek.20210823/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "GPGPU kernel DSL for OCaml"
+maintainer: ["mathias.bourgoin@gmail.com"]
+authors: ["Mathias Bourgoin"]
+license: "Cecill-B"
+homepage: "https://github.com/mathiasbourgoin/SPOC"
+bug-reports: "https://github.com/mathiasbourgoin/SPOC/issues"
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "spoc"
```

Comment:
```suggestion
  "spoc" {= version}
```
Is this meant to always be released together with spoc? If so, a constraint might make maintenance easier in the future and less confusing for users.